dpctl-capi/cmake/modules/FindLevelZero.cmake

```diff
@@ -26,7 +26,11 @@
 # TODO: Add a way to record the version of the level_zero library
 
 find_library(LEVEL_ZERO_LIBRARY ze_loader HINTS $ENV{L0_LIB_DIR})
```

Comment:
Should we also set L0_LIB_DIR in bld.bat? Is it needed from level-zero repo, or what?

Reply:
We access `ze_loader` library dynamically. I do not know why we search for it. @diptorupd Do you have something to comment?

Reply:
The idea was that FindLevelZero.cmake should not really care whether we use the library dynamically. Maybe someone is already doing a more comprehensive cmake module for Level zero than I want to write. I will look if there is one.